components/inputs/input-search.js

```diff
@@ -31,7 +31,7 @@ class InputSearch extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement)))
 			 */
 			disabled: { type: Boolean },
 			/**
-			 * REQUIRED: Accessible label for the input
+			 * ACCESSIBILITY: REQUIRED: Acts as the primary label for the input. Not visible.
```

Comment:
So this is mostly just because of how the documentation repo looks for the accessibility field. It's currently set to look for [`ACCESSIBILITY:`](https://github.com/BrightspaceUI/documentation/blob/main/pages/_components/demo-tables.js#L24) (with the colon). So if we were to want to change how we do things, it would first have to be in there. My guess is that it was done so it could be easier to remove the whole string without also risking it removing other use cases of the word Accessibility (though I guess the all uppercase is probably enough for that)

Reply:
Gotcha. Yeah probably.

Do we have properties that are `ACCESSIBILITY` but not `REQUIRED`?

Maybe we could support both `ACCESSIBILITY:` and `ACCESSIBILITY, REQUIRED:` (whichever order we prefer).

Reply:
I believe we do have some that are either but not both. I'm good with supporting that.

Reply:
Is the syntax entirely up to us? Could it just look for comma-separated values inside square brackets or squiggly brackets for instance? (I'm good with whatever though)

Reply:
I believe it's up to us, and that it just needs to be unique enough to avoid falsely identifying a description if it happens to use the word as part of it's actual description.

Reply:
I'm going to keep this as-is for the time being until the documentation repo is ready to take it so that it doesn't break the current use of it, hope that's alright?

Reply:
Do people have a preference between `ACCESSIBILITY: REQUIRED:` and `REQUIRED, ACCESSIBILITY:`? I think I prefer the later, but I suppose it doesn't really matter. Or something else? I think this is purely for developers looking at the code - the ACCESSIBILITY piece gets removed when the docs are generated.

Regardless of which we choose, a couple of small changes would be required to the docs generation code. My preference is that we decide on what we want, make that small change, and then follow suit here before getting much further. But that doesn't need to block this PR... we already have a few that would need to be cleaned up.

Reply:
++